kubespawner/proxy.py

```diff
@@ -56,6 +59,17 @@ class KubeIngressProxy(Proxy):
         """,
     )
 
+    ingress = List(
+        trait=Dict,
+        config=True,
+        help="""
+        List of ingress in the form of a dict {'host': 'host.example.fr,
+        'tlsSecret': 'tlsSecretName'}.
+
+        Wildcard might not work, refer to your ingress controler documentation.
+        """,
```

Comment:
```suggestion
        List of ingress in the form of a dict {'host': 'host.example.fr,
        'secret_name': 'host-tls'}.

        Wildcard might not work, refer to your ingress controler documentation.
        """,
```
The tls secret is defined as [secret_name](https://github.com/jupyterhub/kubespawner/pull/406/files#diff-4c2e260a4476d246feddf1d239084673283d54250a7df9cecb79afa5eea459f5R779)